README.md

```diff
@@ -14,6 +14,7 @@
 
 ## Responders
 * [Mopidy](https://github.com/trestrantham/hedwig_mopidy) - Responder for Mopidy
+* [Simple Responders](https://github.com/trestrantham/hedwig_mopidy) - A collection of simple Responders inspired by [hubot-scripts](https://github.com/github/hubot-scripts)
```

Comment:
Looks good. Just change the link from `hedwig_mopidy` to link to the Simple Responders repo and I'll merge it. 🙂